src/pages/explore/dex/_components/ExploreDTokensTabs.tsx

```diff
@@ -0,0 +1,86 @@
+import { Dispatch, SetStateAction } from "react";
+import classNames from "classnames";
+import { useTranslation } from "next-i18next";
+import { useWindowDimensions } from "@hooks/useWindowDimensions";
+
+export function ExploreDTokensTabs({
+  tokenSectionChoice,
+  setTokenSectionChoice,
+}: {
+  tokenSectionChoice: TokenLabels;
+  setTokenSectionChoice: Dispatch<SetStateAction<TokenLabels>>;
+}) {
+  const { t } = useTranslation("page-explore-dex");
+  const dimension = useWindowDimensions();
+  return (
+    <div className="md:card-outline-2 p-[0.5px] rounded-[40px] w-fit mt-8">
+      <div className="flex flex-row md:gap-x-3 gap-x-2 rounded-[40px] md:bg-dark-100 p-1 w-fit">
+        <div className="card-outline-2 md:bg-none p-[0.5px] rounded-[52px]">
+          <button
```

Comment:
nitpick, refactor this into a component, it is repeated 3 times?

Reply:
refactored